sdk/communication/Azure.Communication.PhoneNumbers/src/SipRouting/SipRoutingClient.cs

```diff
@@ -309,7 +423,53 @@ public virtual Response<IReadOnlyList<SipTrunkRoute>> GetRoutes(
             scope.Start();
             try
             {
-                var response = _restClient.GetSipConfiguration(cancellationToken);
+                var response = _restClient.Get(cancellationToken:cancellationToken);
+                return Response.FromValue(response.Value.Routes, response.GetRawResponse());
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Get List of configured <see cref="SipTrunkRoute"/>.
+        /// </summary>
+        /// <param name="expand">Expand enum.</param>
+        /// <param name="cancellationToken">Optional cancellation token.</param>
+        /// <returns>List of configured routes.</returns>
+        public virtual Response<IReadOnlyList<SipTrunkRoute>> GetRoutes(
+            ExpandEnum expand,
```

Comment:
We don't need "expand" parameter for routes, because for routes there is no difference.